<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Table</class>
 <widget class="QWidget" name="Table">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="break_button_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font-size: 18px;
background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #FDEDED;
border-color: #EB3028;
}
QPushButton:pressed{
background-color: #F7B7B6;
border-color: #EB3028;
}</string>
       </property>
       <property name="text">
        <string>Сбросить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="result_label">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Результат</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="count_button_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font-size: 18px;
background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #EDFDF0;
border-color: #10DA20;
}
QPushButton:pressed{
background-color: #CCFAD2;
border-color: #10DA20;
}</string>
       </property>
       <property name="text">
        <string>Подсчитать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Save_result_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font-size: 18px;
background-color: #e2e2e2;
border:1px solid #afafaf;
}
QPushButton:hover{
background-color: #E0F2FF;
border-color: #80C8FF;
}
QPushButton:pressed{
background-color: #C2E4FF;
border-color: #80C8FF;
}</string>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="judges_2" stretch="0,0,0,0,0">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="judge_1_l">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="judge_2_l">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="judge_3_l">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="judge_4_l">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="judge_5_l">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
